src/clj/puppetlabs/services/jruby/jruby_core.clj

```diff
@@ -183,6 +184,12 @@
   [jruby-instance :- (schema/pred jruby-schemas/jruby-instance?)]
   @(:state jruby-instance))
 
+(schema/defn register-event-handler
+  "Register the callback function by adding it to the event callbacks atom."
+  [event-callbacks :- Atom
+   callback-fn :- IFn]
+  (swap! event-callbacks conj callback-fn))
```

Comment:
This part of this namespace is our "public" API, right?  If so, do we expect users to be passing around the `event-callbacks` Atom?